blogs/2025/07/15/copilot-coding-agent.md

```diff
@@ -0,0 +1,89 @@
+
+
+# Announcing Copilot Coding Agent Integration in Visual Studio Code
+
+Today, we’re excited to give you a first look at the Copilot Coding Agent in Visual Studio Code.
+
+This is a big step - instead of only having one agent in VS Code, you can now have multiple agents running at once working on your behalf. This is _actually_ being a 10x developer. It's officially a a thing. You get to hand off any work you need done to to a squad of AI teammates that never get tired and will actually read the whole issue before starting work.
```

Comment:
```suggestion
This is a big step - instead of only having one agent in VS Code, you can now have multiple agents running at once working on your behalf. This is _actually_ being a 10x developer. It's officially a thing. You get to hand off any work you need done to a squad of AI teammates that never get tired and will actually read the whole issue before starting work.
```

Reply:
Removed